### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by talal**

### CODE REVIEW

File: gui/src/gui/ssh.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def __init__(self, *args, **kwargs):
         self.stack.add_named(self.preferences_page, "preferences_page")
 
         self.add_ssh_key_group = Adw.PreferencesGroup()
-        self.add_ssh_key_group.set_title("Add an SSH Key")
+        self.add_ssh_key_group.set_title("Add a SSH Key")
```
Comment (quexten) [inline gui/src/gui/ssh.py:40]: "an SSH Key" is actually correct here, because it's pronounced ess-ess-aitch.

**Review by quexten — CHANGES_REQUESTED** [review]
Thanks for the PR! If you revert the SSH change, I'm happy to merge.

File: gui/src/gui/ssh.py (hunk 1 above)
Comment (talal) [reply gui/src/gui/ssh.py]: Ahh, I see. I've reverted this change.

**Comment by talal** [comment]
There are still some typos but since they concern variable names therefore I haven't changed them. I have only corrected typos in user facing code: error strings, GUI, CLI flag description, etc.

You might want to take a look at [typos](https://github.com/crate-ci/typos) and integrate it in your codebase as a GitHub action so that future typos will be automatically flagged.

**Comment by quexten** [comment]
Thanks, great suggestion, added this as an issue in #154

**Comment by quexten** [comment]
Looks good, thanks again for the PR!

**Pull request merged**